external/src/main/java/edu/illinois/cs/cogcomp/pipeline/common/ExternalToolsConfigurator.java

```diff
@@ -32,7 +32,7 @@
      * individually, then splice the annotations together. This allows for partial annotation of
      * documents in cases where document text causes local problems for individual annotators.
      */
-    public static final Property USE_SENTENCE_PIPELINE = new Property("useSentencePipeline", FALSE);
+    public static final Property USE_SENTENCE_PIPELINE = new Property("useSentencePipeline", TRUE);
```

Comment:
doesn't this set the sentence-based pipeline by default? based on the pr name and the description, this seems like the opposite of what you want.